repository_service_tuf_api/api/targets.py

```diff
@@ -48,3 +48,23 @@ def delete(
     _user=Security(validate_token, scopes=[SCOPES_NAMES.delete_targets.value]),
 ):
     return targets.delete(payload)
+
+
+@router.post(
+    "/publish/",
+    summary=(
+        "Submit a task to publish targets."
+        f"Scope: {SCOPES_NAMES.write_targets.value}"
+    ),
+    description=(
+        "Trigger a task to publish targets not yet published from the "
+        "RSTUF Database"
+    ),
+    response_model=targets.Response,
+    response_model_exclude_none=True,
+    status_code=status.HTTP_202_ACCEPTED,
+)
+def post_publish_targets(
```

Comment:
Why not call this method `publish_targets` instead?
Same for `targets.post_publish_targets()` below.

Reply:
I won't be changing it now because I'm following to have the method name in the functions and its is nothins exposed to the users.

But, there is an old issue about refactoring this functions name. (I will start renaming the issue from July 2022)
- #17

Reply:
Yea, but why add new methods to be refactored when this is a new feature not been introduced? 
Do you follow a certain format now?

Reply:
Because we need to decide on the format/standard and apply it consistently, we have this issue to continue the discussion there.
It will not have a significant improvement as this is internal functionality. It doesn't solve or introduce a technical-debit problem refactoring this name and also doesn't unblock our roadmap.
A small PR can solve it with a defined standard for naming the FastAPI functions for endpoints.

Reply:
Okay, let's follow in a discussion about similar changes there in this issue.